src/nvim/screen.c

```diff
@@ -969,7 +982,8 @@ static void win_update(win_T *wp, Providers *providers)
                && (wp->w_topline < wp->w_lines[0].wl_lnum
                    || (wp->w_topline == wp->w_lines[0].wl_lnum
                        && wp->w_topfill > wp->w_old_topfill)
-                   )) {
```

Comment:
I've seen more than one case like this. Is uncrustify doing this by itself? I hope/assume, if the code had been correct to start with i e `)) {` added to the end of the `&& wp->w_topfill > wp->w_old_topfill)`, then uncrustify would _not_ distort it into this.

Reply:
Nah, that's on me. I got 2 less clint errors for some reason when I made this change and I didn't think too much of it. Changing it back :) .

Reply:
Should be fixed now. Please confirm that it looks alright.

Reply:
I assume this has been resolved. Unresolve this if I'm mistaken.